sample/src/main/res/layout/fragment_shopping_cart.xml

```diff
@@ -77,15 +76,37 @@
         android:text="@string/total_price"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/total_price"/>
+        app:layout_constraintTop_toBottomOf="@id/total_price" />
 
     <View
+        android:id="@+id/items_container"
         android:layout_width="match_parent"
         android:layout_height="@dimen/items_container_height"
         android:layout_margin="@dimen/margin_10"
         android:background="@color/white"
         android:elevation="@dimen/margin_2"
-        app:layout_constraintBottom_toTopOf="@+id/checkout_button"/>
+        app:layout_constraintBottom_toTopOf="@+id/checkout_button" />
+
+    <com.polidea.cockpit.sample.shoppingcart.CartItem
+        android:id="@+id/shoes"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:elevation="@dimen/margin_2"
```

Comment:
I would move those items to `items_container` and remove `android:elevation` attribute from each item